common/decisions/hfe_decision_features.txt

```diff
@@ -113,10 +204,15 @@ hfe_decision_features_exodus = {
 				create_ship = {
 					name = "New Hope"
 					random_existing_design = colonizer
+					graphical_culture = hfe_graphical_culture_pirate
```

Comment:
i dont think graphical culture pirate have model for colony ship.
Why you puting changes without testing it?

Reply:
Yes they do. They look like a rod with six balls sticking out of it. That's the image they use for ALL civilian ships.